### DISCUSSION TITLE
Difference between 1 element and 2 elements

### DISCUSSION THREAD

**Discussion by wei0py** [deepmodeling/deepmd-kit#1782] [Q&A]
Hi,

If I have two models, one is for 1 element (A) and another is for 2 element (A and B). (Starting with same input file except for the type_map). But the element (A) in common has same parameters for the embedding net and fitting net in these two models.
Could I know if these two models predict same results for the element in common? If not, could I know where is changed? 

Thanks

**Comment by wanghan-iapcm** [comment]
In general the models for the two systems do not give exactly the same prediction on the shared element. 

If you have trained the models with consistent data, the predictions should be close enough (smaller than the training error).
